src/shared/components/custom-input/input.component.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react'
+import { InputPropsTypes } from '@/shared/types/input.types'
+
+const CustomInput: FC<InputPropsTypes> = ({
```

Comment:
Why your component name and file name differs? Both needs to be same

Reply:
ok